stage1/init/common/kvm_mount.go

```diff
@@ -81,13 +81,14 @@ func MountSharedVolumes(root string, appName types.ACName, volumes []types.Volum
 		}
 
 		destination := filepath.Join(appRootfs, mntPath)
-		// TODO: verify if rkt should do this, or it should be some outer responsibility
-		// to ensure if such patch exists
-		if err := ensureDestinationExists(source, destination); err != nil {
+		// TODO: check if symlinked source tries to escape root
```

Comment:
evaluateAppMountPath is already invocated few lines above, but it's called for `destination`.
I have a limited understanding of https://github.com/coreos/rkt/blob/master/tests/rkt_mount_test.go#L68 test - how anything can escape in this situation root? (this test is failing on kvm even after this patch).